controller/workitemtype.go

```diff
@@ -120,12 +120,7 @@ func (c *WorkitemtypeController) List(ctx *app.ListWorkitemtypeContext) error {
 			return jsonapi.JSONErrorResponse(ctx, errs.Wrap(err, "Error listing work item types"))
 		}
 		// Remove "planneritem" from the list of WITs
-		witModels := []workitem.WorkItemType{}
-		for _, wit := range witModelsOrig {
-			if wit.ID != workitem.SystemPlannerItem {
-				witModels = append(witModels, wit)
-			}
-		}
+		witModels := stripBaseWorkItemTypes(witModelsOrig)
```

Comment:
That was decided by @aslakknutsen . I think we want to keep the repository as agnostic to our UI needs as possible.